client/src/styles.scss

```diff
@@ -1,13 +1,12 @@
-
 // Custom Theming for Angular Material
+@use '~@angular/material' as mat;
```

Comment:
Does the comment right above this still make sense here? There's no mention of theming in this import anymore, so I'm not sure.

Reply:
Hrm, yes, it could do with being reworded, but it's not wrong, either. I'd be inclined to leave it.

Good catch, though!